react/ui-components/src/atoms/BreadCrumb.js

```
@@ -46,32 +54,39 @@ const BreadCrumb = (props) => {
               key={ci}
               style={props?.itemStyle}
-              className={`digit-bread-crumb--item ${"back-crumb-item"}`}
+              className="digit-bread-crumb--item back-crumb-item"
             >
               <span onClick={() => window.history.back()}>{crumb.content}</span>
             </li>
           );
+
         return (
-          <>
+          <Fragment>
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)